src/context/ThemeProvider.res

```diff
@@ -19,7 +19,7 @@ let newDefaultConfig: HyperSwitchConfigTypes.customStylesTheme = {
       primary: "#006DF9",
       secondary: "#F7F7F7",
       sidebar: "#242F48",
-      background: "#F7F8FB",
+      background: "#006df9",
```

Comment:
is there a reason that primary and background both correspond to same hex code ?

Reply:
can we try to merge primary and background both if both are same ?

Reply:
I don't think so , In our previous implementation, we had a separate background field that corresponded to bg-primary in Tailwind.When we introduced the new themes response, it began using a new background field that defaults to bg-background in Tailwind. The default value was not updated to the color getting used.